linux/install-centos7-apache.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e -u -x
+
+OMEROVER=omero
```

Comment:
I'm a little concerned about this sequence here. While it might make some sense it breaks the pattern used in all the other install scripts.